tests/browser_type_test.go

```diff
@@ -18,3 +22,46 @@ func TestBrowserTypeConnect(t *testing.T) {
 	b := bt.Connect(tb.wsURL, nil)
 	b.NewPage(nil)
 }
+
+func TestBrowserTypeLaunchToConnect(t *testing.T) {
+	wsURLEnvVar := "K6_BROWSER_WS_URL"
+
+	setUp := func(wsURL string) {
+		os.Setenv(wsURLEnvVar, wsURL) //nolint:errcheck,gosec
+	}
+	tearDown := func() {
+		os.Unsetenv(wsURLEnvVar) //nolint:errcheck
+	}
```

Comment:
There is [`SetEnv`](https://pkg.go.dev/testing#T.Setenv) for this:

```suggestion
	t.SetEnv(wsURLEnvVar, wsURL)
```

Reply:
Great! Didn't know that, thanks! :+1: 
Ammending this to the original commit and adding you as co-author as more changes are required.
See [here](https://github.com/grafana/xk6-browser/blob/5d42c87d06684561b97f8bb9fa3213b47ea0b8b3/tests/browser_type_test.go#L30-L32).

Reply:
That's why I like code reviews :) To learn from each other's knowledge.